iree/tools/iree-benchmark-module-main.cc

```diff
@@ -182,8 +224,10 @@ int main(int argc, char** argv) {
       "iree-benchmark-module \n"
       "    --module_file=module.vmfb\n"
       "    --entry_function=exported_function_to_benchmark\n"
+      "      if entry_function is not set, all the exported functions will be\n"
+      "      benchmarked and they are expected to not have input arguments\n"
       "    [--function_inputs=2xi32=1 2,1x2xf32=2 1 | "
-      "     --function_inputs_file=file_with_function_inputs]\n"
+      "--function_inputs_file=file_with_function_inputs]\n"
```

Comment:
still want the spaces here for indentation

Reply:
I add a "\n" above, this was causing weird format in `--help`, eg, `[ xxx |          xxx]`.